scripts/basic-workflow-tests.sh

```diff
@@ -31,11 +31,6 @@ BOLD='\033[1m'
 NC='\033[0m' # No Color
 
 SCRIPTDIR=$(dirname "$0")
-export E2E_TEST_DIR
```

Comment:
Why are these variables being removed? We need them to exist for this end-to-end test.